assets/wizards/advertising/views/providers/index.js

```diff
@@ -29,11 +56,21 @@ const Providers = ( { services, toggleService } ) => {
 				actionText={ google_ad_manager && google_ad_manager.enabled && __( 'Configure' ) }
 				toggle
 				toggleChecked={ google_ad_manager && google_ad_manager.enabled }
-				toggleOnChange={ value => toggleService( 'google_ad_manager', value ) }
-				titleLink={ google_ad_manager ? '#/google_ad_manager' : null }
-				href={ google_ad_manager && '#/google_ad_manager' }
+				toggleOnChange={ value => {
+					toggleService( 'google_ad_manager', value ).then( () => {
+						if (
+							value === true &&
+							! google_ad_manager.status.connected &&
+							! google_ad_manager.status.network_code
+						) {
+							setIsOnboarding( true );
+						}
+					} );
+				} }
+				titleLink={ google_ad_manager?.enabled ? '#/google_ad_manager' : null }
+				href={ google_ad_manager?.enabled && '#/google_ad_manager' }
 				notification={
-					google_ad_manager.status.error
+					google_ad_manager.enabled && google_ad_manager.status.error
 						? [ google_ad_manager.status.error ]
```

Comment:
Great idea! 622e1ed2b04c6ad814ef579ca8096a936c1f3f52 implements it a clean up a little bit of how the notifications are handled. I'm using 'connect' instead of 'reconnect' because we are not differentiating if the it has been connected before.